server/src/data_store.cpp

```diff
@@ -250,14 +242,13 @@ bool DataStore::ReadDistinctMacAddrs(const std::string& device_id, std::vector<s
         else
         {
             console_->error("Failed to read from database");
```

Comment:
@platisd modified to one return point only. But, I don't get the need for changing the debug print. 

:sleeping:  Tomorrow is another day :smile:

Reply:
Have a look at the comment right below.